app/Services/GroupService.php

```
@@ -0,0 +1,175 @@
+            throw new Exception("Failed to update group: " . $e->getMessage());
+        }
+    }
+
+    /**
+     * Delete a group
+     *
+     * @param int $id
+     * @return bool
+     * @throws Exception
+     */
+    public function delete($id)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)